lib/middleware/parameter.js

```diff
@@ -209,6 +209,14 @@ module.exports = async (ctx, next) => {
                 });
                 await Promise.all(tasks);
             }
+
+            // scihub
+            if (ctx.query.scihub) {
+                ctx.state.data.item.map((item) => {
+                    item.link = item.doi ? `https://sci-hub.tw/${item.doi}` : `https://sci-hub.tw/${item.link}`;
```

Comment:
should get the direct link for the pdf file

Reply:
Meanwhile 'sci-hub.tw' has been block by some ISP, which will slow down the rendering of local rss reader.

Reply:
> should get the direct link for the pdf file

sci-hub sometimes prompts a captcha, unless that can be addressed, it's not feasible.

Reply:
> Meanwhile 'sci-hub.tw' has been block by some ISP, which will slow down the rendering of local rss reader.

I will add an ENV for defining the scihub mirror link.